packages/firebase_auth/firebase_auth/lib/src/firebase_auth.dart

```diff
@@ -23,15 +27,18 @@ class FirebaseAuth extends FirebasePluginPlatform {
     _delegatePackingProperty ??= FirebaseAuthPlatform.instanceFor(
       app: app,
       pluginConstants: pluginConstants,
+      persistence: _persistence,
     );
+    _persistence = null;
```

Comment:
Is it normal that we set the _persistence to null here?

Reply:
I reset as it was only needed on the first call to initialize auth. But I guess it doesn't matter reseting if we only ever set Auth once once when we call delegate getter:

```dart
 FirebaseAuthPlatform get _delegate {
    _delegatePackingProperty ??= FirebaseAuthPlatform.instanceFor(
      app: app,
      pluginConstants: pluginConstants,
      persistence: _persistence,
    );
    _persistence = null;
    return _delegatePackingProperty!;
  }
```

I've removed it here and on `firebase_auth_web`.

Reply:
Nice thanks :)